packages/core/client.js

```diff
@@ -54,24 +55,45 @@ class BugsnagClient {
     this.BugsnagSession = BugsnagSession
   }
 
-  configure (opts = {}) {
-    this.config = config.mergeDefaults({ ...this.config, ...opts }, this.configSchema)
-    const validity = config.validate(this.config, this.configSchema)
+  setOptions (opts) {
+    this._opts = { ...this._opts, ...opts }
```

Comment:
My JS is rusty, but would calling `setOptions` combine the passed in parameters with the existing values in `this._opts`. Is there any scenario where this might not be desirable?

Reply:
Or some scenario where duplicate options are passed in?

Reply:
this merges whatever you pass in with whatever exists on `this._opts`. Passing in the same key multiple times means the last one will take precendence. In reality I think this will only ever get called once, but I wanted to be on the defensive side of things in case it would be useful to get called again with only a subset of options.